packages/discord.js/src/structures/MessagePayload.js

```diff
@@ -192,7 +192,9 @@ class MessagePayload {
       content,
       tts,
       nonce,
-      embeds: this.options.embeds?.map(embed => new MessageEmbed(embed).toJSON()),
+      embeds: this.options.embeds?.map(embed =>
+        (embed instanceof MessageEmbed ? embed : new MessageEmbed(embed)).toJSON(),
```

Comment:
```suggestion
        (embed instanceof MessageEmbed ? embed.toJSON() : new MessageEmbed(embed)).toJSON(),
```

Reply:
Ig it should also become toJSON()

Reply:
That's not how JavaScript works

Reply:
lol sorry I'm a mad not gonna comment again without properly checking 😢. I didn't notice these ()